File: Tutorial5-1Dagger-Android/src/main/java/com/example/tutorial5_1dagger_android/MainActivity.java
package com.example.tutorial5_1dagger_android;

import android.os.Bundle;
import android.widget.TextView;

import androidx.appcompat.app.AppCompatActivity;

import javax.inject.Inject;

public class MainActivity extends AppCompatActivity {

    @Inject
    DummyDependency dummyDependency;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        TextView textView = findViewById(R.id.textView);
        textView.setText(dummyDependency.getApplicationName());


    }
}

File: Tutorial5-1Dagger-Android/src/main/java/com/example/tutorial5_1dagger_android/MainActivity.kt
package com.example.tutorial5_1dagger_android

import android.os.Bundle
import android.widget.TextView
import dagger.android.AndroidInjection
import dagger.android.support.DaggerAppCompatActivity
import javax.inject.Inject

class MainActivity : DaggerAppCompatActivity() {

    @Inject
    lateinit var dummyDependency: DummyDependency

    override fun onCreate(savedInstanceState: Bundle?) {

        AndroidInjection.inject(this)

        super.onCreate(savedInstanceState)

        setContentView(R.layout.activity_main)

        findViewById<TextView>(R.id.textView).text = "dummyDependency: $dummyDependency"
    }
}